F3FChrono/data/Event.py

```diff
@@ -135,8 +135,10 @@ def add_existing_round(self, f3f_round):
         self.rounds.append(f3f_round)
         if self.current_round is None:
             self.current_round = 0
+            self.current_round_display = 1
```

Comment:
Why is this additional member required ?
It seems, it will be self.current_round + 1 ?

Reply:
I forget to delete this variable..

done.